packages/terra-filter-view/src/_Frame.module.scss

```diff
@@ -0,0 +1,181 @@
+@import '~terra-mixins';
+
+:local {
+  .select-parent {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+  }
+
+  .select {
+    background-color: var(--terra-filter-select-background-color, #fff);
+    color: var(--terra-filter-select-color, #1c1f21);
+    display: inline-flex;
+    font-size: var(--terra-filter-select-font-size, 1.143rem);
+    line-height: var(--terra-filter-select-line-height, 1.5);
+    position: relative;
+  }
+
+  .display {
+    overflow: hidden; // IE 10
+  }
+
+  .button {
+    border: var(--terra-filter-search-button-border, 0.071em solid #dedfe0);
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0.25em;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0.25em;
+    display: inline-block;
+    flex: 0 0 auto;
+    margin: 0;
+
+    &:hover {
+      background-color: var(--terra-filter-search-button-hover-background-color, #b9bbbc);
+      border-color: var(--terrra-filter-search-button-hover-border-color, #c8cacb);
+    }
+
+    &:active {
+      background-color: var(--terra-filter-search-button-active-background-color, #6f7477);
+      border-color: var(--terra-filter-search-button-active-border-color, #52585b);
+      color: #fff;
+    }
+  }
+
+  .clear-button {
+    color: var(--terra-filter-clear-button-color, #b9bbbc);
+    margin-left: 0.3571428571rem;
+    margin-right: 0.3571428571rem;
+    position: absolute;
+    right: 30px;
+    top: 7px;
+  }
+
+  .search-input,
+  .search-input:focus {
+    background: transparent;
+    border: 0;
+    color: inherit;
+    flex: 1 1 auto;
+    font-size: var(--terra-filter-search-font-size, 1.143rem);
+    height: 1.375em;
+    margin-top: var(--terra-filter-search-margin-top, 0.21428571428571427rem);
+    max-width: 100%; // IE 10
+    min-width: 0; // IE 11
+    opacity: 1;
+    outline: none;
+    text-overflow: ellipsis;
+    width: auto;
+
+  }
+
+  .right-padding {
+    // Add padding to avoid text from hiding under clear-search button
+    padding: 0 26px 0 0;
```

Comment:
Similar to the comment above, a flex solution might solve a few of these static positioning issues.